reviews/incubation-crossplane.md

```diff
@@ -0,0 +1,259 @@
+adoption, growth, and maturity of the project.  It has also enabled an ecosystem of solution focused
+“configurations” to be built and shared with the community, removing some of the obstacles with
+getting started and productive building your own platform with Crossplane.
+
+The installation and management of providers (e.g. provider-aws) and configurations in Crossplane
+has greatly improved with the second iteration of Crossplane’s package manager.  The package author
+can express the dependencies it has on other packages, as well as version constraints, so that all
+required packages (and their versions) are installed automatically.  Packages can also be
+automatically upgraded and rolled back to ease adoption and roll out of new versions.
+
+The Crossplane community held a very successful Community Day event to coincide with the release of
+v1.0.  The speaker line up was quite impressive, including the [keynote
+panel](https://www.youtube.com/watch?v=WGfYlssfIIk&list=PL510POnNVaaZx1QDIn_-77igM1COuHaHV&index=2)
+of Kubernetes creators Joe Beda, Brian Grant, and Brendan Burns, which was moderated by Kelsey
+Hightower.  Many adopters of the project also gave talks on their specific use cases and success
+stories with Crossplane, while project maintainers led deep dives through the various feature areas
+of the project.
+
+We’ve also seen impressive growth in the community statistics, indicating greater adoption and a
+growing ecosystem of people getting involved with the project.  Many stats have increased by at
+least 50% in the last 9 months since being accepted into Sandbox, such as the number of GitHub stars
+(~2,000 to 3,100+), commits to the project (~2,000 to 3,100+), and contributors (~50 to ~75+).
+Other metrics have seen greater relative increases, such as container downloads (10x) and Slack
+members (2.5x).  One particular area of growth that we’re most excited about is that we have started
+to receive non-trivial feature contributions from the greater community, such as [bi-directional
+patching](https://github.com/crossplane/crossplane/pull/2077) of composite resources, and
+[composition patchSets](https://github.com/crossplane/crossplane/pull/1972), which indicates a more
+serious adoption and commitment on their part.
+
+### Project and Community Statistics
+
+| Statistic | Sandbox | Current |
+|-|-|-|
+| GitHub stars | 2,120 | 3,100+ |
+| Container downloads | 1,440,000 | 12,600,000+ |
+| Releases | 24 | 32 |
+| Commits | 2,215 | 3,100+ |
+| Contributors (commits) | 48+ contributors | 72+ contributors from 66+ companies |
+| Maintainers | 10 maintainers from 3 orgs | 15 maintainers from 5 orgs |
+| Twitter followers | 1,484 | 2,258+ |
+| Slack members | 610 | 1,696+ |
+| Youtube subscribers | 215 | 524+ |
+
+## Incubation Requirements
+
+### Document that it is being used successfully in production by at least three independent end users which, in the TOC’s judgement, are of adequate quality and scope
```

Comment:
Could you please separately list out end users according to this definition? https://github.com/cncf/toc/blob/main/FAQ.md#what-is-the-definition-of-an-end-user

Reply:
I've now split this production user section into end users and cloud native vendors, as per the CNCF's definition you linked to harry :)